edx_solutions_api_integration/utils.py

```diff
@@ -485,6 +488,27 @@ def get_image_dimensions(image_url):
         return img.size
 
 
+def get_non_actual_company_users(exclude_type, organization_id):
+    """
+    excludes users which are not part of an actual organization
+    """
+    cache_key = u"edx_solutions_api_integration.{category}.{category_type}.{organization_id}".format(
```

Comment:
You can use `get_cache_key` for constructing cache key.

Reply:
Addressed